azkaban-common/src/test/java/azkaban/executor/SelectorTest.java

```diff
@@ -602,4 +605,101 @@ public void testSelectorListWithNullValue() throws Exception {
     Assert.assertTrue(null == executor);
 
   }
+
+  @Test
+  public void testCreatingExectorfilterObject() throws Exception{
+    List<String> validList = new ArrayList<String>(ExecutorFilter.getAvailableFilterNames());
+    try {
+      new ExecutorFilter(validList);
+    }catch (Exception ex){
+      Assert.fail("creating ExecutorFilter with valid list throws exception . ex -" + ex.getMessage());
+    }
+  }
+
+  @Test
```

Comment:
Looks like the junit Expected exception is useful here - http://www.mkyong.com/unittest/junit-4-tutorial-2-expected-exception-test/